spec/Store.spec.js

```diff
@@ -826,12 +824,16 @@ describe('Store', () => {
 
   describe('fetchAll', () => {
     it('always fetches the records with the given type from the server', async () => {
-      expect.assertions(5)
+      expect.assertions(7)
       fetch.mockResponse(mockAllTodosResponse)
       const todos = await store.fetchAll('todos')
+
       expect(todos).toHaveLength(2)
       expect(todos[0].title).toBe('Do taxes')
+      expect(todos[0].id).toBe('1')
```

Comment:
`fetchAll` wasn't setting `id`. I'm not sure how this was even working, although I suspect it may have something to do with the fact that many our our serializes (erroneously) serialize the id.